.bcr/metadata.template.json

```diff
@@ -0,0 +1,23 @@
+{
+  "homepage": "https://github.com/protocolbuffers/protobuf",
+  "maintainers": [
+    {
+      "email": "sandyzhang@google.com",
+      "github": "zhangskz",
+      "name": "Sandy Zhang"
+    },
+    {
+      "email": "mkruskal@google.com",
+      "github": "mkruskal-google",
+      "name": "Mike Kruskal"
+    },
+    {
+      "email": "gberg@google.com",
+      "github": "googleberg",
+      "name": "Jerry Berg"
+    }
+  ],
+  "repository": ["github:protocolbuffers/protobuf"],
+  "versions": [],
```

Comment:
Should this be propagated with the current versions published to BCR?

  "versions": [
    "3.19.0",
    "3.19.2",
    "3.19.6",
    "21.7",
    "23.1"
  ],
  "yanked_versions": {
    "3.19.0": "CVE-2022-3171 (https://github.com/advisories/GHSA-h4h5-3hr4-j3g2)",
    "3.19.2": "CVE-2022-3171 (https://github.com/advisories/GHSA-h4h5-3hr4-j3g2)"
  }

Reply:
no, this file is the template to be filled in by the Publish to BCR app